laravel-app/resources/views/auth/login.blade.php

```diff
@@ -4,14 +4,20 @@
 <div class="container">
     <div class="row justify-content-center">
         <div class="col-md-4">
-            <h1 class="text-center p-5">Microblog</h1>
+            <div class="d-flex justify-content-center align-items-center text-center p-4">
+                <img class="img-fluid w-25" src="{{ asset('microblog-logo.png') }}" alt="Microblog Logo">
+                <h1 class="ml-0">icroblog</h1>
+            </div>
+
+            <!-- Login card -->
```

Comment:
I was wondering maybe you could create a component for a card.

https://laravel.com/docs/8.x/blade#rendering-components